src/overlays/actors/ovl_Obj_Kendo_Kanban/z_obj_kendo_kanban.c

```diff
@@ -1,28 +1,40 @@
 /*
  * File: z_obj_kendo_kanban.c
  * Overlay: ovl_Obj_Kendo_Kanban
- * Description: Swordsman's School - Cuttable Board
+ * Description: Swordsman's School - Cuttable Board at back of school
  */
 
 #include "z_obj_kendo_kanban.h"
 #include "objects/object_dora/object_dora.h"
+#include "z64math.h"
```

Comment:
This should already be included from "global.h" in the actors header, no need to include it again.